### ISSUE TITLE
Custom Instrument families/templates

### ISSUE THREAD

**Issue by TimBomBom** [musescore/MuseScore#25623]
### Your idea

So, I envision being able to create custom profiles/families of instruments when creating new scores (we have the default options like Common, Orchestral, Concert Band, Pop/Rock, etc). This would allow users to add or remove instruments they more or less commonly work with. The user would be able to set a profile they like (custom or stock) as their default family that will be selected when creating a new score (I believe the default now is Common). Perhaps the creation menu of said custom families will be like adding instruments to a score, but on the left column it shows ALL instruments, and the right column is what will go in the family.

### Problem to be solved

A user may have a particular style and instrumentation they mainly follow, but not all the instruments may be in the Common family (i.e. english horn is not by default in the woodwinds tab), however if they switch to a different family that does contain that instrument, it may be clutterred with instruments they'd never use, or be missing others they would (i.e. switching to the Orchestral family would let them add the english horn to their score, but then the saxophones would be missing, and the screen is filled with instruments like basset horn and contrabass clarinet that the user doesn't want to use). It can quickly become annoying to have to filter through various groups of instruments to go on hunts for the instruments you want. This would allow users to comfortably add/remove instruments from their personal common rotation as they choose, and also create profiles for the various genres they dabble in. This could extend to saving custom templates as well (like in the "Create score from template" tab).

### Additional context

![custom family suggestion](https://github.com/user-attachments/assets/d3e189f9-71cf-4736-989b-ef54a114b19d)

I created a horribly rough sketch of what I'm talking about above.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by SteveBlower** [comment]
Indeed, "common" is rather subjective.  That list may be common for person A but a different list would be common for person B.  

A "Favourites" selection at the top of the groups or immediately available outside the grouping structure would be nice.
